<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SFEditDlg</class>
 <widget class="QWidget" name="SFEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>SF Values</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDoubleSpinBox" name="minValSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>displayed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="maxValSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QxtSpanSlider" name="dispValSlider">
        <property name="toolTip">
         <string>Min displayed value</string>
        </property>
        <property name="statusTip">
         <string>Min displayed value</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="minSatSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>color sat.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="maxSatSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QxtSpanSlider" name="satValSlider">
        <property name="toolTip">
         <string>Min saturation value</string>
        </property>
        <property name="statusTip">
         <string>Min saturation value (color gradient start)</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="absSatCheckBox">
          <property name="text">
           <string>absolute saturation</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="logScaleCheckBox">
          <property name="text">
           <string>log scale</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="releaseBoundariesCheckBox">
          <property name="toolTip">
           <string>Release boundaries (let the user defines them)</string>
          </property>
          <property name="text">
           <string>release boundaries</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="nanInGreyCheckBox">
          <property name="text">
           <string>show NaN in grey</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>frame</zorder>
  <zorder>releaseBoundariesCheckBox</zorder>
  <zorder>releaseBoundariesCheckBox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxtSpanSlider</class>
   <extends>QSlider</extends>
   <header>qxtspanslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
